src/Pages/ProductList/ProductList.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import ProductItem from "./../../Components/ProductItem/ProductItem";
+import FilterSelectBox from "./Components/FilterSelectBox/FilterSelectBox";
+import "./ProductList.scss";
+
+class ProductList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      product: [],
+      isClick: false,
+      styleList: [
+        "loose leaf tea",
+        "loose leaf tisane",
+        "powdered tisane",
+        "sugar",
+        "teabag",
+      ],
+      typeOfTea: [
+        "black breakfast tea",
+        "black tea",
+        "chinese black tea",
+        "Chinese green tea",
+        "flavored black tea",
+        "flavored green tea",
+        "flavored oolong tea",
+        "flavored white tea",
+        "fruit tisane",
+        "herbal tisane",
+        "japanese green tea",
+        "japanese matcha",
+        "oolong tea",
+        "premium green tea",
+        "smoked black tea",
+        "sugar tisane",
+        "Tea",
+        "white Tea",
+      ],
+      price: ["Under $10", "Under $20", "Under $30", "Under $40", "Under $50"],
+      sortBy: [
+        "Sort by:",
+        "Sort A-Z",
+        "Price Descending",
+        "Price Ascending",
+        "Top Rated",
+        "Best Sellers",
+      ],
+    };
+  }
+
+  componentDidMount = () => {
+    fetch("http://localhost:3000/Data/product/mockProduct.json")
```

Comment:
```suggestion
    fetch(`${API}/Data/product/mockProduct.json`)
```
지금은 mockdata를 쓰고 있지만 통신을 하게 되면 반드시 루트 폴더에 config.js를 만들고 해당 경로에서 api 주소를 일괄적으로 관리해주세요!